app/controllers/specialist_documents_controller.rb

```diff
@@ -46,4 +46,8 @@ def base_path
   def error_403(exception)
     render plain: exception.message, status: 403
   end
+
+  def content_store
+    @content_store ||= GdsApi::ContentStore.new(Plek.current.find("content-store"))
+  end
```

Comment:
Why is this being added? As far as I can tell, without this change, `content_store` is a method included from `GdsApi::Helpers`, that does mostly the same thing as this.